.github/workflows/cla.yml

```diff
@@ -10,9 +10,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: 'CLA Assistant'
-        if: github.base_ref != 'main' && ((github.event.comment.body == 'recheckcla' || github.event.comment.body == 'I have read the CLA Document and I hereby sign the CLA') || github.event_name == 'pull_request_target')
-        # Alpha Release
-        uses: gnosis/github-action@master
+        if: github.base_ref != 'main' && (/I have read the CLA Document and I hereby sign the CLA|recheckcla/.test(github.event.comment.body) || github.event_name == 'pull_request_target')
```

Comment:
Also changed the strict equality check to a regex to account for insignificant newlines or some such.